src/main/java/gregtech/api/recipes/machines/FuelRecipeMap.java

```diff
@@ -34,6 +38,13 @@ public FuelRecipeMap(String unlocalizedName) {
         return RECIPE_MAPS;
     }
 
+    @ZenMethod
+    public static FuelRecipeMap getByName(String name) {
```

Comment:
That parameter should be `unlocalizedName` so it is consistent with RecipeMap and with filter